projects/golang/go/1.15/patches/0101-Don-t-use-the-bundled-tzdata-at-runtime-except-for-t.patch

```diff
@@ -1,4 +1,4 @@
-From edce31a2904846ae74e3c011f2cf5fddc963459e Mon Sep 17 00:00:00 2001
```

Comment:
Can we not change these? I am assuming its a side effect of the way we apply the patches triggers the shas to change?